src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -107,6 +107,8 @@
                 "alert": "Es muss mindestens eine Option (Offline-IDE oder Online-Editor) ausgewählt sein"
             },
             "offlineIde": "Offline-IDE",
+            "irisActivated": "Iris Chatbot",
+            "irisActivatedTooltip": "Durch Aktivierung dieser Checkbox wird der Iris-Chatbot für Studierende freigeschaltet.",
```

Comment:
Instructors should get a bit more information what the chatbot does. You can also include a jhi-documentation-button once there is a documentation about Iris